docs/sources/administration/user-management/manage-org-users/index.md

```diff
@@ -74,7 +74,7 @@ If you have [server administrator]({{< relref "../../roles-and-permissions/#graf
 
 ## Invite a user to join an organization
 
-When you invite users to join an organization, you assign the **Admin**, **Editor**, or **Viewer** role which controls user access to the dashboards and data sources owned by the organization. Users receive an email that prompts them to accept the invitation.
+When you invite users to join an organization, you assign the **Admin**, **Editor**, **Viewer**, or **None** role which controls user access to the dashboards and data sources owned by the organization. Users receive an email that prompts them to accept the invitation.
```

Comment:
Nit: I think we're referring to frontend workflows in this guide, and from the frontend we call this "No basic role"
```suggestion
When you invite users to join an organization, you assign the **Admin**, **Editor**, or **Viewer** role, or select **No basic role**. Organization roles control user access to resources, such as dashboards and data sources, owned by the organization. Users receive an email that prompts them to accept the invitation.
```